scenarios/kubernetes_e2e.py

```diff
@@ -402,14 +402,14 @@ def build_kops(kops, mode):
     check('make', 'gcs-publish-ci', 'VERSION=%s' % version, 'GCS_LOCATION=%s' % gcs)
 
 
-def set_up_aws(args, mode, cluster, runner_args):
+def set_up_aws(workspace, args, mode, cluster, runner_args):
     """Set up aws related envs."""
     for path in [args.aws_ssh, args.aws_pub, args.aws_cred]:
         if not os.path.isfile(os.path.expandvars(path)):
             raise IOError(path, os.path.expandvars(path))
     mode.add_aws_cred(args.aws_ssh, args.aws_pub, args.aws_cred)
 
-    aws_ssh = '/workspace/.ssh/kube_aws_rsa'
+    aws_ssh = os.path.join(workspace, '.ssh', 'kube_aws_rsa')
```

Comment:
oh whoops - this $WORKSPACE is local workspace, we need to use the workspace mapped into docker container.

Reply:
is this the cause of the kops failure?

Reply:
It does appear to be the cause. 😕 Fixed in https://github.com/kubernetes/test-infra/pull/4730, thanks @krzyzacy